add how to contribute (#8129)
PyTorchLightning/pytorch-lightning: README.md

The lightning community is maintained by
- [10+ core contributors](https://pytorch-lightning.readthedocs.io/en/latest/governance.html) who are all a mix of professional engineers, Research Scientists, and Ph.D. students from top AI labs.

> - 480+ community active contributors.

Reviewer: ```suggestion
- 480+ active community contributors.
```

> - read how you can [start contributing](https://devblog.pytorchlightning.ai/quick-contribution-guide-86d977171b3a)

Reviewer 2: Why not add the content directly in the docs instead?

Author: then we can add it on both places, but many people reds just readme

> - read how you can [start contributing](https://devblog.pytorchlightning.ai/quick-contribution-guide-86d977171b3a)

Reviewer: ```suggestion
- read about how you can [start contributing](https://devblog.pytorchlightning.ai/quick-contribution-guide-86d977171b3a)
```

> - read about how you can [start contributing](https://devblog.pytorchlightning.ai/quick-contribution-guide-86d977171b3a)

Reviewer 3: this can't really be a bullet point under "the lightning community is maintained by ..."

The lightning community is maintained by
- [10+ core contributors](https://pytorch-lightning.readthedocs.io/en/latest/governance.html) who are all a mix of professional engineers, Research Scientists, and Ph.D. students from top AI labs.
- 480+ active community contributors.

> Want to join us? Read about how you can [start contributing](https://devblog.pytorchlightning.ai/quick-contribution-guide-86d977171b3a)

Reviewer 4: ```suggestion
Want to help us build Lightning and reduce boilerplate for thousands of researchers? [Learn how to make your first contribution here](https://devblog.pytorchlightning.ai/quick-contribution-guide-86d977171b3a)
```
